lambdas/utils/lloyd_george_validator.py

```diff
@@ -87,3 +96,38 @@ def check_for_file_names_agrees_with_each_other(file_name_list: list[str]):
     ]
     if len(set(expected_common_part)) != 1:
         raise LGInvalidFilesException("File names does not match with each other")
+
+def validate_with_pds_service(file_name_list: list[str], nhs_number: str):
+    try:
+    # Check file names match with the nhs number in metadata.csv
```

Comment:
this comment is outdated so probably better to change it or delete it.

Reply:
Done